setup-nexus-npm/README.md

```diff
@@ -1,22 +1,20 @@
 # setup-nexus-npm
 
-Configure a github action to point towards a self hosted Npm repository. After usage, any calls to `npm install` will point towards the repository at the endpoint provided.
+A composite github action for creating a .npmrc file that will point to the npm repositories in nexus
 
 ## Parameters
-- Endpoint: the npm registry endpoint
-- Login: the login for the self hosted npm repository, typically stored as a secret
-- Password: the password for the self hosted npm repository, typically stored as a secret
+- username: the username for accessing the npm repositories in nexus, typically provided as a secret from the calling workflow
+- password: the password for accessing the npm repositories in nexus, typically provided as a secret from the calling workflow
 
 ## Example Usage
 
 ```yaml
 - uses: actions/setup-node@v3
   with:
-    node-version: 14
-- name: Set up setup-nexus-npm
+    node-version: 16
```

Comment:
Any reason we don't jump to 18 here as it's the latest long term release?  I know GHA runs on Node16 right now, but if we're setting up Node in our own runner I'd assume that we can use whichever we'd like.

Reply:
no, I just looked at the version of node I was running and it was 16.  18 it is...